portfolio_ahardman.html

```diff
@@ -0,0 +1,118 @@
+				<circle cx='178.75' cy='30' r='4'/>
+				<circle cx='231.25' cy='30' r='4'/>
+				<circle cx='283.75' cy='30' r='4'/>
+				<circle cx='336.25' cy='30' r='4'/>
+				<circle cx='388.75' cy='30' r='4'/>
+				<circle cx='441.25' cy='30' r='4'/>
+				<circle cx='493.75' cy='30' r='4'/>
+				<circle cx='546.25' cy='30' r='4'/>
+				<circle cx='598.75' cy='30' r='4'/>
+				<circle cx='651.25' cy='30' r='4'/>				
+				<circle cx='703.75' cy='30' r='4'/>
+				<circle cx='756.25' cy='30' r='4'/>
+				<circle cx='808.75' cy='30' r='4'/>
+				<circle cx='861.25' cy='30' r='4'/>
+				<circle cx='913.75' cy='30' r='4'/>
+				<circle cx='966.25' cy='30' r='4'/>
+			</svg>
+			<div class='link container'>
+				<a href='#'>Home</a>
+				<a href='#portfolio'>Portfolio</a>
+				<a href='#about_me'>About</a>
+				<a href='#contact'>Contact</a>
+			</div>
+	</div>
+	<main id='main'>
+		<div class='container'>
+			<h1>Amanda Hardman</h1>
+			<h3>Fullstack Developer</h3>
+		</div>
+	</main>
+	<section id='portfolio' class='portfolio'>
+		<div class='container'>
+			<h2>Portfolio</h2>
+			<div class='builds_container'>
+				<div class='build box1'>
+					(Incomplete)
+					<a href='#' class='popup webpage'>Demo Webpage</a>
+					<a href='#' class='popup github'>GitHub</a>
+				</div>
+				<div class='build box2'>
+					(Incomplete)
+					<a href='https://htmlpreview.github.io/?https://github.com/amandahrdmn/pilot-webpage-exercise/blob/master/ahardman_pilot_page.html' class='popup webpage'>Demo Webpage</a>
+					<a href='https://github.com/amandahrdmn/pilot-webpage-exercise' class='popup github'>GitHub</a>
+				</div>
+				<div class='build box_empty'>
+					Coming Soon!
+				</div>
+				<div class='build box_empty'>
+					Coming Soon!
+				</div>
+				<div class='build box_empty'>
+					Coming Soon!
+				</div>
+				<div class='build box_empty'>
+					Coming Soon!
+				</div>
+			</div>
+		</div>
+	</section>
+	<section id='about_me' class='about_me'>
+		<div class='container'>
+			<h2>About Me</h2>
+			<div class='p_container'>
+			<p>I'm currently studying at Mayden Academy and hope to further my web development skills and progress to a junior dev role and specialise in back-end development.</p>
+				<p>In my spare time I enjoy knitting and cross stitch.</p>
+				<p>Have a look around and contact me if you'd like more details!</p>
+			</div>
+		</div>
+	</section>
+	<footer id='contact' class='contact'>
+		<div class='center container'>
+			<h3>Get in touch via:</h3>
+			<div class='right container'>
+				<form class='email'>
+					<input class='email_input' type='email' placeholder='Email Address'>
+					<button type='submit'>Subscribe</button>
+				</form>
+				<a class='linkedin' href='https://www.linkedin.com/in/amanda-hardman-69399469/'><img src='https://image.flaticon.com/icons/svg/174/174857.svg' alt='linked in logo'/></a>
+				<a href='https://github.com/amandahrdmn'><img src='https://image.flaticon.com/icons/svg/25/25231.svg' alt='github logo'/></a>
```

Comment:
indentations like this might make your img tags easier to find/read.

```suggestion
				<a href='https://github.com/amandahrdmn'>
				          <img src='https://image.flaticon.com/icons/svg/25/25231.svg' alt='github logo'/>
				</a>
```